sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1459,6 +1459,29 @@ def _path_residuals(
 class LinearModelCV(MultiOutputMixin, LinearModel, ABC):
     """Base class for iterative model fitting along a regularization path."""
 
+    _parameter_constraints = {
+        "eps": [Interval(Real, 0, None, closed="left")],
+        "n_alphas": [Interval(Integral, 0, None, closed="left")],
+        "alphas": [np.ndarray, None],
```

Comment:
`alphas` an also accept lists. In general we define it for "array-like" objects. There's an `"array-like"` helper constraint. Note some estimators, e.g. `ElasticNetCV` define it only for `np.ndarray` in their docstring, but they still accept array-likes and the docstring should be updated.
```suggestion
        "alphas": ["array-like", None],
```